Sources/VaporJWT/Request+JWT.swift

```diff
@@ -0,0 +1,50 @@
+import HTTP
+import JWT
+
+extension Request {
+    /// Parses and returns a JWT from the request
+    /// if one exists.
+    /// The JWT will also be verified with the supplied
+    /// signer.
+    public func jwt(verifyUsing signer: Signer) throws -> JWT {
```

Comment:
I like your point about preventing access to unverified JWTs. If we also provide a go-to way to deal with claims I also don't see much need to access the JWT at all.
Another thought would be to make this method internal and provide a public `var verifiedJWT: JWT?` on `Request` that retrieves a stored version of the JWT (or nil) to prevent duplicate verification.